app/views/admin/dashboard/index.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="adminDashboardContainer">
+  <section class="activeOrders">
+    <% @orders.by_status.each do |order| %>
+      <div id="order_<%= order.id %>">
+       <p><%= link_to "#{order.name}", "/admin/users/#{order.user.id}" %></p>
```

Comment:
I would change this to display the user's name instead of the name put on the order since they _can_ be different, and it sounds like the user story wants the user's name
```suggestion
       <p><%= link_to "#{order.user.name}", "/admin/users/#{order.user.id}" %></p>
```